lib/models/get_repositories/repos_result_rto.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import 'repository_rto.dart';
+
+part 'repos_result_rto.g.dart';
+
+@JsonSerializable(fieldRename: FieldRename.snake)
+class ReposResultRTO {
```

Comment:
I would always try to write everything out in full like Repos being Repositories

Reply:
done